### ISSUE TITLE
Search for playlist with url

### ISSUE THREAD

**Issue by Felixnightshade** [vfsfitvnm/ViMusic#116]
### Feature description

If a playlist URL is searched for, the app should open it or at least show it in the search results.

### Why do you want this feature?

It is currently impossible to add unlisted playlists. Also, playlists set to public often don't show up in the search results across YouTube. This disproportionately affects playlists from new users and significantly harms the new user experience.

### Additional information

A smoother method could be to have an entry that leads to the playlist on click in the top slot of the search auto complete list if a valid URL is detected.

**Comment by vfsfitvnm** [comment]
It is already possible. What's the playlist url?

**Comment by Felixnightshade** [comment]
For example:
https://m.youtube.com/playlist?list=PLdPV7DRKXdJF9MJpb6ghgirQYssB_s7vE

What URL _does_ work for you?

**Comment by vfsfitvnm** [comment]
Yeah. I totally forgot to handle `m.youtube` links there. However, you should be able to open that like this:
1) Paste the link somewhere (e.g. Telegram)
2) Open the link and select "Open with ViMusic"

**Comment by Felixnightshade** [comment]
I see. Works great! But while we're at it, this URL form also does not work. 

https://music.youtube.com/playlist?list=PLdPV7DRKXdJF9MJpb6ghgirQYssB_s7vE&feature=share

**Comment by vfsfitvnm** [comment]
That url shohld work (it didn't in `v0.2.0`), are you using `v0.2.1`?

**Comment by Felixnightshade** [comment]
I am using the latest version from fdroid, which is v0.2.0. So I suppose that is the issue.
